db/python/tables/sequencing_group.py

```diff
@@ -69,15 +78,37 @@ async def query(
             'platform': 'sg.platform',
             'active_only': 'NOT sg.archived',
             'external_id': 'sgexid.external_id',
+            'created_before': 'DATE(sg.row_start)',
+            'created_on': 'DATE(sg.row_start)',
+            'created_after': 'DATE(sg.row_start)',
+            'assay_meta': 'sga_subquery.assay_meta',
         }
 
         wheres, values = filter_.to_sql(sql_overrides)
         _query = f"""
-            SELECT {self.common_get_keys_str}
-            FROM sequencing_group sg
+            SELECT 
+                {self.common_get_keys_str}, 
+                JSON_ARRAYAGG(sga_subquery.assay_meta) AS assay_meta,
+                TIMESTAMP(sg.row_start) as created_on
+            FROM sequencing_group FOR SYSTEM_TIME ALL AS sg
             LEFT JOIN sample s ON s.id = sg.sample_id
             LEFT JOIN sequencing_group_external_id sgexid ON sg.id = sgexid.sequencing_group_id
+            LEFT JOIN (
+                SELECT
+                    sequencing_group_id,
+                    assay_subquery.meta AS assay_meta
+                FROM
+                    sequencing_group_assay
+                    LEFT JOIN (
+                        SELECT
+                            id,
+                            meta
+                        FROM
+                            assay
+                    ) AS assay_subquery ON sequencing_group_assay.assay_id = assay_subquery.id
+            ) AS sga_subquery ON sg.id = sga_subquery.sequencing_group_id
             WHERE {wheres}
+            GROUP BY sg.id
```

Comment:
`GROUP BY` works here because there is a one-many mapping when joining the sg table to assay table that is needed to aggregate the assay meta into a list using the `JSON_ARRARAGG` function, but if `assay_meta` is removed as per the above comment then we can remove `GROUP BY`, unless it's required for the [timestamp computation](https://github.com/populationgenomics/metamist/pull/615#discussion_r1413353144) as you suggested?